catalog/app/data/workflows.spec.js

```diff
@@ -0,0 +1,108 @@
+import parseWorkflows, {
+  emptyWorkflowsConfig,
+  workflowNotAvaliable,
+  workflowNotSelected,
+} from './workflows'
+
+describe('Workflows config', () => {
+  it('empty file', () => {
+    const config = parseWorkflows('')
+    expect(config).toBe(emptyWorkflowsConfig)
```

Comment:
iirc, `.toBe` check for reference equality, but semantically it makes more sense to check for structural / value equality here (`.toEqual` iirc)

Reply:
Yes, it's reference equality, and I wanted to check it for reference equality

Reply:
but i don't think that's right, bc it doesn't matter if it returns the exact object defined in the module (it's an implementation detail), what really matters here is the value / structure of the returned object